docs/docusaurus.config.js

```diff
@@ -177,6 +187,27 @@ async function createConfig() {
                 },
               ],
             },
+            {
+              title: 'Company',
+              items: [
+                {
+                  label: 'Imprint',
+                  to: 'https://www.edgeless.systems/imprint',
+                },
+                {
+                  label: 'Privacy Policy',
+                  href: 'https://www.edgeless.systems/privacy',
+                },
+                {
+                  html: `<a href="javascript: Cookiebot.renew()" class="footer__link-item">Cookie Settings</a>`
+                },
+
+                {
+                  label: 'Contact Us',
+                  href: 'https://www.edgeless.systems/contact/',
```

Comment:
Is there any rule when you use `to` or `href`? If not, you can choose one of these:
* use `to` only for pages on docs.edgeless.systems, `href` for anything else
* use `to` for all pages on edgeless.systems, `href` for all external

The only difference seems to be that the href links get that arrow icon appended.
You'd also need to adapt some of the existing links.

Reply:
Yes, thanks for noticing. the first one was intended.